packages/consensus/src/validation/block.cairo

```
@@ -69,6 +68,9 @@ pub fn compute_and_validate_tx_data(
         txids.append(txid);
 
         if (is_coinbase) {
+            if (is_coinbase_txid_duplicated(txid, block_height)) {
+                continue;
```

We should throw an error here I think